pmoves/services/channel-monitor/channel_monitor/main.py

```
@@ -49,9 +49,24 @@ def _parse_scopes(value: str | None) -> list[str]:
     google_scopes=GOOGLE_SCOPES or None,
 )
 
+@asynccontextmanager
```

Ruff (0.14.10)

Undefined name `asynccontextmanager`

(F821)